tests/topotests/static_routing_with_ebgp/test_static_routes_topo2_ebgp.py

```diff
@@ -0,0 +1,2338 @@
+
+    sleep(5)
+
+    step("Random no shut of the nexthop interfaces")
+    # Bringup interface
+    shutdown_bringup_interface(tgen, dut, intf, True)
+
+    step(
+        "After random shut/no shut of nexthop , only that "
+        "nexthop deleted/added from all the routes , other nexthop remain "
+        "unchanged"
+    )
+    dut = "r2"
+    protocol = "static"
+    for addr_type in ADDR_TYPES:
+        input_dict_4 = {
+            "r2": {
+                "static_routes": [
+                    {
+                        "network": PREFIX1[addr_type],
+                        "next_hop": NEXT_HOP_IP["nh" + str(nhp)][addr_type],
+                    }
+                ]
+            }
+        }
+    result = verify_rib(
+        tgen,
+        addr_type,
+        dut,
+        input_dict_4,
+        next_hop=nh_all[addr_type],
+        protocol=protocol,
+    )
+    assert result is True, "Testcase {} : Failed \nError: Routes are"
+    " missing in RIB".format(tc_name)
+
+    step("Remove random static route with all the nexthop")
+    dut = "r2"
+    randnum = random.randint(1, 7)
+    for addr_type in ADDR_TYPES:
+        input_dict_4 = {
+            "r2": {
+                "static_routes": [
+                    {
+                        "network": PREFIX1[addr_type],
+                        "next_hop": NEXT_HOP_IP["nh" + str(randnum)][addr_type],
+                        "delete": True,
+                    }
+                ]
+            }
+        }
+        logger.info("Configure static routes")
+        result = create_static_routes(tgen, input_dict_4)
+        assert result is True, "Testcase {} : Failed \n Error: {}".format(
+            tc_name, result
+        )
+
+        step(
+            "After delete of random route , that route only got deleted from"
+            " RIB/FIB other route are showing properly"
+        )
+        nh = NEXT_HOP_IP["nh{}".format(randnum)][addr_type]
+        result = verify_rib(
+            tgen,
+            addr_type,
+            dut,
+            input_dict_4,
+            next_hop=nh,
+            protocol=protocol,
+            expected=False,
+        )
+        assert result is not True, "Testcase {} : Failed \nError: Routes are"
+        " missing in RIB".format(tc_name)
+
+    for addr_type in ADDR_TYPES:
+        input_dict_4 = {
+            "r2": {
+                "static_routes": [
+                    {
+                        "network": PREFIX1[addr_type],
+                        "next_hop": NEXT_HOP_IP["nh" + str(randnum)][addr_type],
+                    }
+                ]
+            }
+        }
+        logger.info("Configure static routes")
+        result = create_static_routes(tgen, input_dict_4)
+        assert result is True, "Testcase {} : Failed \n Error: {}".format(
+            tc_name, result
+        )
+
+    step("Reload the FRR router")
+    # stop/start -> restart FRR router and verify
+    stop_router(tgen, "r2")
+    sleep(5)
```

Comment:
here too

Reply:
Changes made are:
1. Removed the static sleep.
2. Base config is modified to use bgp aggressive timers.